packages/netlify/package.json

```diff
@@ -35,7 +35,8 @@
     "@astrojs/underscore-redirects": "^0.3.4",
     "@netlify/functions": "^2.8.0",
     "@vercel/nft": "^0.27.4",
-    "esbuild": "^0.21.5"
+    "esbuild": "^0.21.5",
+    "vite": "^5.4.2"
```

Comment:
we no longer need `vite` in deps since we vendor the search code

Reply:
Not quite, the PR includes https://github.com/withastro/adapters/pull/381/files#diff-a2534d55417999ca927cbdb98267b817468cec23ba870f9f70a7077710d797bbR6

Reply:
Ah right, I thought the vendor happened in both packages